homeassistant/components/unifi/__init__.py

```diff
@@ -64,8 +64,8 @@ async def async_setup_entry(hass, config_entry):
         configuration_url=controller.api.url,
         connections={(CONNECTION_NETWORK_MAC, controller.mac)},
         default_manufacturer=ATTR_MANUFACTURER,
-        default_model="UniFi Controller",
-        default_name="UniFi Controller",
+        default_model="UniFi Network Application",
```

Comment:
Naming has become a bit inconsistent, can we use UniFi Network Controller in all places where we have Controller or Application?
```suggestion
        default_model="UniFi Network Controller",
```

Reply:
My intent was to mirror the new branding, but I kind of did decided on it a bit late (I am really bad at being consistent, lol). 

The add-on store now uses "Network Application" instead of "Controller": https://github.com/hassio-addons/addon-unifi

As well as the getting started page on ui.com: https://help.ui.com/hc/en-us/articles/1500012237441-UniFi-Network-Use-the-UniFi-Network-Application

That is inline with the the forum post I mentioned and linked in the docs.

Reply:
Also, I do not think _all_ of the comments are going to be able to be 100% clean on application vs. controller because so many of the classes / variables / etc. still use the controller terminology. I think the most important bit is to at least try to get the user facing strings.

Reply:
Just going with controller in the code and then application for user facing stuff might be the best approach for consistency until / if you want to refactor the code (which I am trying to avoid in this PR to keep it "small").

Reply:
Sorry, been busy with the Protect stuff. We are getting closer to merging our UniFi Protect integration into core.

After thinking about it more, while the new marketing branding is "UniFi Network Application" and the old stuff was "UniFi Controller", the difference between controller/application can be really confusing. So I went through and dropped controller/application everywhere so now it is just "UniFi Network".